packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -1349,218 +1349,64 @@ describe('NetworkController', () => {
-          await withController(
-            {
-              infuraProjectId: 'infura-project-id',
-            },
-            async ({ controller }) => {
-              const fakeInfuraProvider = buildFakeInfuraProvider();
-              createInfuraProviderMock.mockReturnValue(fakeInfuraProvider);
-              const fakeInfuraSubprovider = buildFakeInfuraSubprovider();
-              SubproviderMock.mockReturnValue(fakeInfuraSubprovider);
-              const fakeMetamaskProvider = buildFakeMetamaskProvider([
-                {
-                  request: {
-                    method: 'eth_chainId',
-                  },
-                  response: {
-                    result: '0x1337',
-                  },
-                },
-              ]);
-              createMetamaskProviderMock.mockReturnValue(fakeMetamaskProvider);
-
-              await controller.setProviderType(networkType);
-
-              expect(createInfuraProviderMock).toHaveBeenCalledWith({
-                network: networkType,
-                projectId: 'infura-project-id',
-              });
-              expect(createMetamaskProviderMock).toHaveBeenCalledWith({
-                dataSubprovider: fakeInfuraSubprovider,
-                engineParams: {
-                  blockTrackerProvider: fakeInfuraProvider,
-                  pollingInterval: 12000,
-                },
-              });
-              const { provider } = controller.getProviderAndBlockTracker();
-              const promisifiedSendAsync = promisify(provider.sendAsync).bind(
-                provider,
-              );
-              const chainIdResult = await promisifiedSendAsync({
-                method: 'eth_chainId',
-              });
-              expect(chainIdResult.result).toBe('0x1337');
-            },
-          );
-        });
-
-        it('updates networkDetails.isEIP1559Compatible in state based on the latest block (assuming that the request for eth_getBlockByNumber is made successfully)', async () => {
-          const messenger = buildMessenger();
-          await withController(
-            {
-              messenger,
-            },
-            async ({ controller }) => {
-              const fakeInfuraProvider = buildFakeInfuraProvider();
-              createInfuraProviderMock.mockReturnValue(fakeInfuraProvider);
-              const fakeInfuraSubprovider = buildFakeInfuraSubprovider();
-              SubproviderMock.mockReturnValue(fakeInfuraSubprovider);
-              const fakeMetamaskProvider = buildFakeMetamaskProvider([
-                {
-                  request: {
-                    method: 'eth_getBlockByNumber',
-                    params: ['latest', false],
-                  },
-                  response: {
-                    result: {
-                      baseFeePerGas: '0x1',
-                    },
+    [NetworkType.mainnet, NetworkType.goerli, NetworkType.sepolia].forEach(
```

Comment:
I'd strongly recommend hiding whitespace when looking at this diff. This line is basically the only one with changes, the rest are indentation/lint changes.